Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -10611,6 +10634,11 @@
 			package = 606754B728CF8A190033CD5E /* XCRemoteSwiftPackageReference "facebook-ios-sdk" */;
 			productName = FacebookLogin;
 		};
+		606C45FB29FAD091001BA067 /* FirebaseRemoteConfig */ = {
```

Comment:
Shouldn't see this...we're using the existing Firebase dependency for now.

Reply:
So the firebase package is only added to the main target. In order for these new files that have been added to the Library target to access remote config (` RemoteConfig.remoteConfig()`) we need to add FirebaseRemoteConfig to the same target